packages/ds-medicare-gov/docs/src/pages/components/Button/Button.docs.scss

```diff
@@ -0,0 +1,96 @@
+/*
+Button
+
+@uswds https://designsystem.digital.gov/components/buttons 
+
+<p class="ds-text--lead">Action buttons allow people to take actions from a current state to a future state. They are promises to the user; they must deliver the promise they offer by doing what the button says it would do.</p>
+
+- PRIMARY CTA is TEAL - Use for the priority action that we want users to take. These actions are required for the user to move to the next step in the user flow (example: Enroll).
+- SECONDARY CTA is NAVY - Use for actions available but not the primary action we want users to take (example: Plan Details). There doesn't have to be a primary button on the page to use the secondary button.
+- TERTIARY CTA is OUTLINE BUTTON with the color that matches the other CTA on the page - Use for actions available but do not take priority over a primary or secondary CTAs (example: Cancel). For example: If primary button is teal, tertiary button has teal outline.
+- Outline CTAs are not used alone - They must be paired with either a primary or secondary CTA.  
+- Exception: CTA over background hero image on landing pages.
+- Lower priority actions are links - Refer to link styling guidelines under Typography.
+- Landing Page Button - This larger button is ONLY used in the hero image of landing pages.
+
+<p>View more guidance and design assets at <a href="https://cms.invisionapp.com/dsm/cms/medicare">Medicare.gov DSM</a>.</p>
+
+<h3>Button classes</h3>
+<ul>
+<li><b>Mobile buttons</b> - Consider using full-width buttons on mobile devices. This will help improve the click range for both left or right-handed users. Use the class <code>mds-c-button--mobile-full-width</code> to apply full-width for mobile and the <a href="/utilities/margin/">responsive margin classes</a> to adjust margins on mobile.</li>
+<li><b>Hiding buttons on print</b> - Hide buttons when printing to avoid unnecessary content for printed material. Use the <code>ds-u-visibility--hidden-print</code> class to hide buttons or any other elements on print. Refer to the <a href="/utilities/display-visibility/">display/visibility utilities</a> for more information.</li>
+</ul>
+
+Markup: button.example.html
+
+Style guide: components.button
+*/
+
+/*
+`<Button>`
+
+The `Button` component accepts its text as children (AKA inner HTML), which
+means you can also pass in HTML or custom components. This gives you a lot of
+flexibility and supports a variety of advanced use cases. The most common use
+case would be passing in an SVG icon along with the text.
+
+In addition to the supported props listed, you can also pass in additional
+props, which will be passed to the rendered root component. For example,
+you could pass in a `target` prop to pass to the rendered anchor element.
+
+@react-example Button.example.tsx
+
+@react-props Button.jsx
```

Comment:
In the build, I'm seeing the error
```
addReactPropData Invalid key Button.jsx for packages/ds-medicare-gov/docs/src/pages/components/Button/Button.docs.scss
```
Is that because in the latest version of core, Button is `Button.tsx`? Should that be updated here?

Reply:
Applied your suggestion, and the error went away! 5e169140136c0d4f6274cd99d151a1b55f5d6198